slides/ocl.md

```diff
@@ -48,7 +48,7 @@ Sometimes, the UML lacks precision. Suppose the following class diagram:
 
 
 - How do you specify that this class only considers people born after 1900?
-- And how do you specify that cycles are not allowed (i.e., that a person cannot be an ancestral of himself)?
```

Comment:
"ancestor". "Ancestral" is an adjective.

Reply:
Ancestor was the word I was suggesting, it was ancestral before.

Reply:
My bad, sorry for this.